plyer/platforms/linux/volume.py

```diff
@@ -0,0 +1,34 @@
+from subprocess import Popen, PIPE
+from plyer.facades import Volume
+from plyer.utils import whereis_exe
+
+from os import environ
+
+
+class VolumeSet(Volume):
+    def _get_state(self):
```

Comment:
Why `_get_state` if you are actually setting the volume and not getting anything out?